index.html

```diff
@@ -19,14 +19,123 @@ <h1 class="main-text">I'M ZALIHA GLAD TO SEE YOU</h1>
+									<span class="stack">Javascript</span>
+								</div>
+								<a  href="#"><h6 class="project-link">See project</h6></a>
 					</div>
-  
+					<div class="works-section">
+							<div>
+								<img class=image-work src="./images/secondimage.png" alt="">
+								<h1>Multi-post Stories</h1>
+								<span class="ball">Canopy</span>
+								<span>Backend</span>
+								<span>2015</span>
+							</div>
+							<div>
+								<p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+									
+										<span class="stack">Html</span>
+										<span class="stack">Css</span>
+										<span class="stack">Javascript</span>
+									</div>
+									<a  href="#"><h6 class="project-link">See project</h6></a>
+						</div>
+						<div class="works-section">
+								<div>
+									<img class=image-work src="./images/thirdimage.png" alt="">
+									<h1>Tonic</h1>
+									<span class="ball">Canopy</span>
+									<span>Backend</span>
+									<span>2015</span>
+								</div>
+								<div>
+									<p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+										
+											<span class="stack">Html</span>
+											<span class="stack">Css</span>
+											<span class="stack">Javascript</span>
+										</div>
+										<a  href="#"><h6 class="project-link">See project</h6></a>
+							</div>
+							<div class="works-section">
+									<div>
+										<img class=image-work src="./images/fouthimage.png" alt="">
+										<h1>Multi-Post Stories</h1>
+										<span class="ball">Canopy</span>
+										<span>Backend</span>
+										<span>2015</span>
+									</div>
+									<div>
+										<p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+											
+												<span class="stack">Html</span>
+												<span class="stack">Css</span>
+												<span class="stack">Javascript</span>
+											</div>
+											<a  href="#"><h6 class="project-link">See project</h6></a>
+								</div>
+				</section>
+				<section class="about">
+				
+						<h1 class="main-text">About Myself</h1>
+						<p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+						<a href="#" class="connect">LET'S CONNECT</a>
+						<div class="icons">
+								<img src="./images/twitter.png" alt="">
+								<img src="./images/Linkedin.png" alt="">
+								<img src="./images/vector-m.png" alt="">
+								<img src="./images/baby.svg" alt="">
+								<img src="./images/hand.png" alt="">
+								
+						</div>	
+						<a  href="#"><h6 class="project-link">GET MY RESUME</h6></a>
+					</section>
+					
+					<section>
+							<div class="language">
+								<h3>Languages</h3>
+								  </div>						
+							<div class="lang">
```

Comment:
- [ ] You should try to use `h1` once per HTML file, check the "About Myself" title.
- [ ] Try to add the margin/padding-right to make it more similar your project to the Figma design you selected.
- [ ] Add down row image after "Languages"
- [ ] Add right row image after "Frameworks" and "Skills".
- [ ] Focus on the size for the "Get my resume" button and the "About Myself" title.
- [ ] [OPTIONAL] You can try to fix the background for JavaScript, HTML, CSS list.

![image](https://user-images.githubusercontent.com/60164239/129459298-ad366800-cfcd-4c70-9bf5-eadc21572186.png)